source/MaterialXGraphEditor/RenderView.cpp

```diff
@@ -806,6 +807,10 @@ void RenderView::renderFrame()
         {
             material->getProgram()->bindUniform(mx::HW::ALPHA_THRESHOLD, mx::Value::createValue(0.99f));
         }
+        if (material->getProgram()->hasUniform(mx::HW::FRAME))
+        {
+            material->getProgram()->bindUniform(mx::HW::FRAME, mx::Value::createValue(static_cast<float>(_frame++)));
```

Comment:
This will update the frame count for every material drawn instead of once per refresh. Same in the other code updating frame rate. Suggest to modify / update frame outside of the refresh logic. This will allow for a proper frame setting (vs refresh count being used as  frame count)
even if not handled with this change.

Reply:
I've moved the increment from during the binding to where the draw call for the frame is made, and added a limit to 32767.